src/routes/(public)/(guest)/login/+page.svelte

```diff
@@ -129,3 +134,9 @@
         </li>
     </svelte:fragment>
 </Unauthenticated>
+
+<style>
+    :global(.auth-provider-button) {
+        margin-bottom: var(--gap-s, 8px);
+    }
```

Comment:
maybe lets just wrap the button for now in a div with this style. don't wanna add more globals. we anyway plan to remove `class` support from Pink because it makes us use `:global`. Same for the other one.